src/app/modules/wallet/pages/accounts/accounts.component.ts

```diff
@@ -110,7 +110,7 @@ export class AccountsComponent {
       let bal = '0';
       let status = 'unknown';
       if (balanceItem) {
-        status = balanceItem.status.toLowerCase();
+        status = balanceItem.status !== '' ? balanceItem.status.toLowerCase() : 'unknown';
```

Comment:
The value is actually undefined, would it be better check for an undefined?    

balanceItem.status !== undefined

Reply:
I got an empty string (See screenshot)... Maybe Browser or OS related. I will check for both (Update it tonight)

Reply:
added !balanceItem.status